bin/setup

```diff
@@ -29,6 +29,10 @@ chdir APP_ROOT do
     end
   end
 
+  step "Installing libmagic, a dependency for ruby-filemagic" do
+    system!("brew install libmagic")
```

Comment:
this is happening after `bundle install` so the gem install of `ruby-filemagic` fails when I run the setup script -- switch up order?

also, do we want to skip install if this is already installed?

Reply:
Good catch! I'll fix this up